torch/csrc/jit/codegen/cuda/lower_validation.cpp

```diff
@@ -235,7 +250,11 @@ void validateVectorize(Fusion* fusion) {
       IterDomain* concrete_id =
           GpuLower::current()->caParallelMap().getConcreteMappedID(id);
 
-      if (concrete_id->getParallelType() == ParallelType::Vectorize) {
+      auto ptype = i <= tv->getComputeAtPosition()
+          ? concrete_id->getParallelType()
+          : id->getParallelType();
```

Comment:
Is this conditional processing necessary? The parallel map should not have a mapping for axes right of the CA position, so `concrete_id == id` when `i > tv->getComputeAtPosition()`.

Reply:
I recall seeing failing cases but don't see them no, so will remove.